components/test_util/src/logging.rs

```diff
@@ -23,6 +24,16 @@ struct CaseTraceLogger {
     f: Option<Mutex<File>>,
 }
 
+// FIXME: Remove this type when slog::Never implements Display.
```

Comment:
Do you have a PR or issue pending upstream for this?

Reply:
Filed slog-rs/slog#209 (`Infallible` will become a `typedef` of `!`, the same as `Never`, and `Infallible` does impl Display).